tests/issues/issue-1618.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+return [
+    'issue-1618-A' => [
+        'ua' => 'Evolution/3.28.0',
+        'properties' => [
+            'Comment' => 'Evolution Generic',
+            'Browser' => 'Evolution',
+            'Browser_Type' => 'Email Client',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'The GNOME Project',
+            'Browser_Modus' => 'unknown',
+            'Version' => '0.0',
+            'MajorVer' => '0',
+            'MinorVer' => '0',
+            'Platform' => 'Linux',
+            'Platform_Version' => 'unknown',
+            'Platform_Description' => 'Linux',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Linux Foundation',
+            'Alpha' => false,
+            'Beta' => false,
+            'Win16' => false,
+            'Win32' => false,
+            'Win64' => false,
+            'Frames' => false,
+            'IFrames' => false,
+            'Tables' => false,
+            'Cookies' => false,
+            'BackgroundSounds' => false,
+            'JavaScript' => false,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'ActiveXControls' => false,
+            'isMobileDevice' => false,
+            'isTablet' => false,
+            'isSyndicationReader' => false,
+            'Crawler' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '0',
+            'AolVersion' => '0',
+            'Device_Name' => 'Linux Desktop',
+            'Device_Maker' => 'unknown',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Linux Desktop',
+            'Device_Brand_Name' => 'unknown',
+            'RenderingEngine_Name' => 'unknown',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'unknown',
+        ],
+        'lite' => false,
+        'standard' => false,
+        'full' => true,
+    ],
+    'issue-1618-B' => [
+        'ua' => 'CamelHttpStream/1.0 Evolution/3.28.0',
```

Comment:
This one too?

Reply:
The 3.28 doesn't exists yet

Reply:
I think you should leave these out of the test suite then.

I don't generally add tests that are picked up by the generic file, unless they're an "invalid version", in which case we're trying to keep those in the `issue-000-invalid-versions.php` file. A "hasn't been released yet" version doesn't really qualify for that though, because once that version is released, the test will have to be modified, which just adds more maintenance.